src/Components/PostCard/PostCard.jsx

```diff
@@ -1,51 +1,49 @@
-import React from 'react'
+import React from "react";
 import * as FaIcons from "react-icons/fa";
 
- const PostCard = () => {
+const PostCard = () => {
   return (
-    <div className="post-card border-2  mt-1">
-              <div className="flex items-center justify-between p-2">
-                <div className="flex items-center">
-                  <img
-                    className="post-avatar mr-2"
-                    src="https://avatars.githubusercontent.com/u/84632214?v=4"
-                    alt=""
-                  /> 
-                  <h1 className="post-user-name">@pritamkr</h1>
-                </div> 
-                <span>
-                  <FaIcons.FaTrash className="icons profile-icons" />
-                </span>
-              </div>
+    <div className="post-card border-2  mt-1 bg-[#fff]">
+      <div className="flex items-center justify-between p-2">
+        <div className="flex items-center">
+          <img
+            className="post-avatar mr-2"
+            src="https://avatars.githubusercontent.com/u/84632214?v=4"
+            alt=""
+          />
+          <h1 className="post-user-name">@pritamkr</h1>
+        </div>
+        <span>
+          <FaIcons.FaTrash className="icons profile-icons" />
+        </span>
+      </div>
 
-              <div className="post-thumbnail-wrapper my-2 p-0 ">
-                <img
-                  src="https://avatars.githubusercontent.com/u/84632214?v=4"
-                  className=""
-                  alt="user-name"
-                />
-              </div>
+      <div className="post-thumbnail-wrapper my-2 p-0 ">
+        <img
+          src="https://avatars.githubusercontent.com/u/84632214?v=4"
+          className=""
```

Comment:
if className is not required, u can remove that

Reply:
ok